x-pack/plugins/cloud_integrations/cloud_links/kibana.jsonc

```diff
@@ -10,7 +10,7 @@
     "optionalPlugins": [
       "cloud",
       "security",
-      "guided_onboarding"
+      "guidedOnboarding"
```

Comment:
Just curious - Do we know when this regression was introduced?

Reply:
Looking at `x-pack/plugins/cloud_integrations/cloud_links/kibana.jsonc` and `src/plugins/ui_actions_enhanced`, it seems the typo has always been here?